views/manage/invites.tt

```diff
@@ -0,0 +1,134 @@
+[%# manage/invites.tt
+  #
+  # Authors:
+  #     Jen Griffin <kareila@livejournal.com>
+  #
+  # Copyright (c) 2023 by Dreamwidth Studios, LLC.
+  #
+  # This program is free software; you may redistribute it and/or modify it under
+  # the same terms as Perl itself.  For a copy of the license, please reference
+  # 'perldoc perlartistic' or 'perldoc perlgpl'.
+  #
+%]
+
+[%- sections.title = '.title' | ml -%]
+
+[%- CALL dw.active_resource_group( "foundation" ) -%]
+
+[%- sections.head = BLOCK %]
+    <style type="text/css">
+        #invites-list {
+            width: 100%;
+        }
+        #invites-list td.invite-info {
+            padding-left: 22px;
+            text-indent: -18px;
+        }
+        .invite-from {
+            color: #777;
+        }
+        .invite-accept,
+        .invite-decline {
+            white-space: nowrap;
+        }
+        #content input[type="radio"] {
+            margin: 0;
+        }
+    </style>
+[%- END -%]
+
+[% IF responses %]
+  [% IF responses.accepted.size %]
+<h2>[% '.head.accepted' | ml %]</h2>
+<p>[% '.body.accepted' | ml %]</p>
+<dl>[% FOREACH row IN responses.accepted %]
+  <dd>[% row.0.ljuser_display %]:
+      [% FOREACH attr IN row.1;
+           IF loop.count != 1; ", "; END; ".label.$attr" | ml;
```

Comment:
Hmm. This took me a bit to figure out what it was doing, so I'm wondering if it might be better to have a more verbose (but clearer) variant here like
```
[%- attrs  = [] -%]
[% FOREACH attr IN row.1;
attrs.push(dw.ml(".label.$attr");
END %]
[% attrs.join(' ,') %]
```
But I'm willing to be overruled on this one. (if only TT had a `map` function, sigh)

Reply:
Heh, if you look at the original BML file, it's concatenating `"item1, item2, item3, "` and then chopping off the last two characters of the string. 🙀